test/unit/specs/components/ImageDetails/v-copy-license.spec.js

```diff
@@ -54,12 +54,12 @@ describe('CopyLicense', () => {
   })
 
   it('should contain the correct contents', () => {
-    const wrapper = render(CopyLicense, options)
+    const wrapper = render(VCopyLicense, options)
     expect(wrapper.find('.copy-license')).toBeDefined()
   })
 
-  it('should dispatch COPY_ATTRIBUTION', () => {
-    const wrapper = render(CopyLicense, options)
+  it.skip('should dispatch COPY_ATTRIBUTION', () => {
```

Comment:
Will we be able to un-skip these tests? If not can we add a comment with a note for why they're skipped and an issue to fix the problem soon?

Reply:
Heck, I was missing why they weren't working with the composition API but it turned out to be something simple in the end. Done!